app/lib/frontend/templates.dart

```diff
@@ -786,6 +786,12 @@ class TemplateService {
         'text': '[outdated]',
         'title': 'Package version too old, check latest stable.',
       });
+    } else if (status == AnalysisStatus.legacy) {
```

Comment:
Does this only happen for packages that don't allow Dart 2 in the Dart SDK constraints? It doesn't happen for packages that do allow Dart 2 but their dependencies don't?

That would be alright, we can tighten the feature later.

There's also packages that do allow Dart 2, but were released prior to Dart 2 and have analysis errors. We could mark those packages as Dart 2 incompatible as well. But that doesn't need to be now.

Reply:
> Does this only happen for packages that don't allow Dart 2 in the Dart SDK constraints?

This PR: yes.

> It doesn't happen for packages that do allow Dart 2 but their dependencies don't?

It could/will happen in:
https://github.com/dart-lang/pub-dartlang-dart/pull/1587